interfaces/builtin/pulseaudio_test.go

```diff
@@ -39,27 +39,42 @@ var _ = Suite(&PulseAudioInterfaceSuite{
 	iface: &builtin.PulseAudioInterface{},
 })
 
+const paMockPlugSnapInfoYaml = `name: other
+version: 1.0
+plugs:
+ pulseaudio:
+  interface: pulseaudio
+apps:
+ app2:
+  command: foo
+  plugs:
+   - pulseaudio
+`
+
+const paMockSlotSnapInfoYaml = `name: pulseaudio
+version: 1.0
+slots:
+ pulseaudio:
+  interface: pulseaudio
+apps:
+ app1:
+  command: foo
+  slots:
+   - pulseaudio
+`
+
+const paMockSlotOSSnapInfoYaml = `name: pulseaudio
```

Comment:
Renamed s/pa/pulseaudio/ in this test. The idea is to match the name of the tested interface (not the  "name:"), so you will now see  "pulseaudioMockPlugSnapInfoYaml = `name: other"... in this file too.